scripts/end_to_end_test.py

```diff
@@ -143,7 +151,12 @@ def get_default_test_inputs(self, workflow_class: Type[BaseWorkflow]) -> Optiona
     def push_workflow_to_vellum(self, workflow_class: Type[BaseWorkflow]) -> bool:
```

Comment:
This method should just accept a module name, and call `vellum workflows push [module]`

Reply:
and we could call this before we even load the workflow class in its own try/except